README.md

```diff
@@ -1,50 +1,70 @@
-![PactSafe Logo](Additional%20Documentation/images/pactsafe-logo.png)
-
-
-
-# iOS SDK
-
-- [Requirements](#requirements)
-- [Installation](#installation)
-- [Notes Before Getting Started](#notes-before-getting-started)
-- [Configure and Initalize the PactSafe SDK](#configure-and-initalize-the-pactsafe-sdk)
-- [PSClickWrapView](#psclickwrapview)
-	- [Preloading Clickwrap Data](#preloading-clickwrap-data)
-	- [Loading Your Clickwrap](#loading-your-clickwrap)
-		- [Interface Builder](#interface-builder)
-		- [Programmatically](#programmatically)
-		- [Configure Contracts Link Tap Behavior](#configure-contracts-link-tap-behavior)
-		- [Check if Checkbox is Selected](#check-if-checkbox-is-selected)
-		- [Sending Acceptance](#sending-acceptance)
-- [Checking Acceptance](#checking-acceptance)
-	- [Using the signedStatus Method](#using-the-signedstatus-method)
-	- [Using the PSAcceptanceViewController](#using-the-psacceptanceviewcontroller)
-	- [Using signedStatus Method and Present Alert](#using-signedstatus-method-and-present-alert)
-- [Sending Activity Manually](#sending-activity-manually)
-- [Customizing Acceptance Data](#customizing-acceptance-data)
-	- [Connection Data](#connection-data)
-	- [Custom Data](#custom-data)
+![Ironclad Logo](Additional%20Documentation/images/ironclad-logo.png)
```

Comment:
was the ironclad logo already added to this repo? didn't see that get added in this PR

Reply:
Good catch @mewelling ! I added this logo. It did not get added on the `git add -u`. Is it because of the space in additional documents? 
<img width="703" alt="image" src="https://github.com/pactsafe/pactsafe-ios-sdk/assets/113391507/75f16800-7373-4f52-93ee-ffe95a9788c7">